ext/oembed/Civi/Oembed/OembedDiscovery.php

```diff
@@ -43,16 +43,24 @@ public function onInvoke(array $path) {
     // In explicit sharing, the administrator sees a panel with options for sharing.
     if (Civi::settings()->get('oembed_share') && \CRM_Core_Permission::check('administer oembed')) {
       $query = $oembed->findPropagatedParams($_GET);
-      $oembed = \Civi::service('oembed')->create($pathStr, $query);
-      $vars = [
-        'oembedSharingUrl' => (string) Civi::url('frontend://civicrm/share', 'a')->addQuery([
-          'url' => (string) Civi::url('frontend://' . $pathStr, 'a')->addQuery($query),
-        ]),
-        'oembedSharingIframe' => $oembed['html'],
+
+      $data = [
+        'page' => Civi::url('frontend://' . $pathStr, 'a')->addQuery($query),
+        'iframe' => Civi::url('iframe://' . $pathStr, 'a')->addQuery($query),
+        'share' => Civi::url('frontend://civicrm/share', 'a')->addQuery(['url' => Civi::url('frontend://' . $pathStr, 'a')->addQuery($query)]),
```

Comment:
Just confirming that frontend://civicrm/share is something that gets maps to a different URL depending on the User Framework/CMS of the producer site, and also allows different entry points for each, specifically WordPress which cannot have a url/route be mapped to file system location at top level within docroot if we are to comply with WP software policies.

Reply:
@JoeMurray, yeah `frontend://civicrm/share` is an ordinary frontend route -- going through the normal `CRM_Core_Invoke`, `CRM_Core_Menu`, `civicrm_invoke()`, etc. It's just a stub.

* It can convey extra params like `?maxwidth` and `?maxheight`.
* If you paste into an oEmbed client, then it's replaced by the `<iframe>`.
* If you open it manually, then it redirects to the real page.